frontend/appflowy_flutter/lib/mobile/presentation/database/board/widgets/mobile_hidden_groups_column.dart

```diff
@@ -150,88 +160,97 @@ class MobileHiddenGroup extends StatelessWidget {
-          childrenPadding: EdgeInsets.zero,
-          title: Row(
-            children: [
-              Expanded(
-                child: Text(
-                  context.read<BoardBloc>().generateGroupNameFromGroup(group),
-                  style: Theme.of(context).textTheme.bodyMedium,
-                  maxLines: 2,
-                  overflow: TextOverflow.ellipsis,
-                ),
-              ),
-              GestureDetector(
-                child: const Padding(
-                  padding: EdgeInsets.all(4),
-                  child: FlowySvg(
-                    FlowySvgs.hide_m,
-                    size: Size.square(20),
-                  ),
-                ),
-                onTap: () => showFlowyMobileConfirmDialog(
-                  context,
-                  title: FlowyText(LocaleKeys.board_mobile_showGroup.tr()),
-                  content: FlowyText(
-                    LocaleKeys.board_mobile_showGroupContent.tr(),
+            return ExpansionTile(
+              tilePadding: EdgeInsets.zero,
+              childrenPadding: EdgeInsets.zero,
+              title: Row(
+                children: [
+                  Expanded(
+                    child: Text(
+                      context
+                          .read<BoardBloc>()
+                          .generateGroupNameFromGroup(group),
+                      style: Theme.of(context).textTheme.bodyMedium,
+                      maxLines: 2,
+                      overflow: TextOverflow.ellipsis,
+                    ),
                   ),
-                  actionButtonTitle: LocaleKeys.button_yes.tr(),
-                  actionButtonColor: Theme.of(context).colorScheme.primary,
-                  onActionButtonPressed: () => context.read<BoardBloc>().add(
-                        BoardEvent.toggleGroupVisibility(
-                          group,
-                          true,
-                        ),
+                  GestureDetector(
+                    child: const Padding(
+                      padding: EdgeInsets.all(4),
+                      child: FlowySvg(
+                        FlowySvgs.hide_m,
+                        size: Size.square(20),
                       ),
-                ),
+                    ),
+                    onTap: () => showFlowyMobileConfirmDialog(
+                      context,
+                      title: FlowyText(LocaleKeys.board_mobile_showGroup.tr()),
+                      content: FlowyText(
+                        LocaleKeys.board_mobile_showGroupContent.tr(),
+                      ),
+                      actionButtonTitle: LocaleKeys.button_yes.tr(),
+                      actionButtonColor: Theme.of(context).colorScheme.primary,
+                      onActionButtonPressed: () =>
+                          context.read<BoardBloc>().add(
+                                BoardEvent.toggleGroupVisibility(
+                                  group,
+                                  true,
```

Comment:
Do we need to send this boolean with the event? Toggle should just reverse the existing state.

Reply:
I renamed this event